Classes/LanguageManager.cpp

```diff
@@ -0,0 +1,91 @@
+#include "LanguageManager.h"
+
+LanguageManager* LanguageManager::_instance = 0;
+
+LanguageManager::LanguageManager()
+{
+	string fileName;
+	// detect current language
+	switch (CCApplication::getInstance()->getCurrentLanguage())
+	{
+	case LanguageType::ENGLISH:
```

Comment:
The language files for the other languages will be the same as English. In the future a language can be added just by replacing the file.